public/stylesheets/style.css

```diff
@@ -48,3 +48,12 @@
   border-radius: 4px; /* future proofing */
   -khtml-border-radius: 4px; /* for old Konqueror browsers */
 }
+
+.btn {
```

Comment:
Overwriting all .btn styles with !important is not the best idea.

Reply:
agreed! It would be better create another class and not overwrite bootstrap's classes

Reply:
Please, make this, so I can merge it

Reply:
@jchabrowski Thank you for the suggestion. I Will make it . @shakaran

Reply:
@jchabrowski You could try just reducing the number of columns that this buttons are using to one instead of two.

<img width="663" alt="image" src="https://github.com/mongo-express/mongo-express/assets/40678306/dc71a721-aa85-4396-b68b-45c9fe60dfd1">